css/src/components/button-close.scss

```diff
@@ -0,0 +1,11 @@
+.button-close {
```

Comment:
Should we rename this to something that doesn't sound like it's a subcomponent of `.button`?

Reply:
Renamed to `.dismiss`